### PR TITLE
Reps-platform 2.0

### PR DESCRIPTION
**Pull request by libetho**

### CODE REVIEW

File: resources/reps-platform.make [hunk 1]
```diff
@@ -8,15 +8,34 @@ core = 7.x
 projects[admin_language][subdir] = "contrib"
 projects[admin_language][version] = 1.0-beta1
 
+projects[admin_theme][subdir] = "contrib"
```
Comment (iulianweb) [inline resources/reps-platform.make]: Needs evaluation.

File: resources/reps-platform.make [hunk 2]
```diff
@@ -8,15 +8,34 @@ core = 7.x
...
+projects[admin_theme][version] = 1.0
+
+projects[administration_language_negotiation][subdir] = "contrib"
```
Comment (iulianweb) [inline resources/reps-platform.make]: Needs evaluation.

File: lib/modules/custom/mapeditor/ec_geocoder/.svn/all-wcprops [hunk 3]
```diff
@@ -0,0 +1,29 @@
+K 25
```
Comment (iulianweb) [inline lib/modules/custom/mapeditor/ec_geocoder/.svn/all-wcprops]: .svn folder should not be here.

**Review by iulianweb — CHANGES_REQUESTED** [review]

**Comment by iulianweb** [comment]
Also please fix all coding standard violations, see attached.
[reps_phpcs.txt](https://github.com/ec-europa/ec-reps-platform/files/1074106/reps_phpcs.txt)

File: resources/reps-platform.make (hunk 1 above)
Comment (libetho) [reply resources/reps-platform.make]: Removed (not used anymore)

File: resources/reps-platform.make (hunk 2 above)
Comment (libetho) [reply resources/reps-platform.make]: https://webgate.ec.europa.eu/CITnet/jira/browse/NEMA-358

Comment (iulianweb) [reply resources/reps-platform.make]: I don't see any proof that someone from the QA team approved this contrib and I am not saying this contrib is not OK...

**Comment by iulianweb** [comment]
Please fix all coding standard violations:
[reps_phpcs_2.txt](https://github.com/ec-europa/ec-reps-platform/files/1087573/reps_phpcs_2.txt)

**Comment by libetho** [comment]
Hi @iulianweb 

Regarding coding standards to fix:
- **reps_admin**: The problematic value is not prefixed by reps_admin because I'm getting de value previously declared in the parent module: reps_core
- **reps_migrate**: I don't think that the remaining issue are blocking because these class are called in country specif modules and all these modules are already deployed. The goal by keeping reps_migrate for the moment is to uninstall it in a clean way as you can see in my update in reps_core.
- **mbp_sync**: this module is developed by @drishu but he is on holidays for the moment. I don't know how I can fix this. If you can help me for this i should be great (I'm available on Lync).

Regarding the module to evaluate can you check this with Mohamed? 
Because the Multisite ticket related to this has been closed without final answer: https://webgate.ec.europa.eu/CITnet/jira/browse/MULTISITE-16915
If the answer is negative I can move back to the admin_language module.

In advance for you help!
[cs-review.txt](https://github.com/ec-europa/ec-reps-platform/files/1090765/cs-review.txt)

**Comment by iulianweb** [comment]
Hi @libetho 
OK for reps_admin, reps_migrate, mbp_sync. False positive should be excluded in the resources/phpcs-custom.xml.
As agreed please go back to the approved module. It seems there are some issues https://webgate.ec.europa.eu/CITnet/jira/browse/REPR-472

File: lib/features/reps_core/reps_core.install [hunk 4]
```diff
@@ -176,3 +173,98 @@ function reps_core_update_7001() {
...
+ * - reps_migrate
+ * - nexteuropa_migrate
+ * - weight.
+ *
+ * Enable the following list of modules:
+ * - extlink
+ * - mbp_sync
+ * - media_browser_plus
+ * - media_bulk_upload
+ * - nexteuropa_metatags
```
Comment (iulianweb) [inline lib/features/reps_core/reps_core.install:196]: Please add rabbit_hole in this list too.

**Comment by libetho** [comment]
Hi @iulianweb ,

Everything should be ok.
Thank for the work and don't hesitate if you have any additional remarks

Regards

**Pull request merged**